src/Engine/Manager/RenderManager.cpp

```diff
@@ -306,7 +306,38 @@ void RenderManager::Render(World& world, const glm::mat4& translationMatrix, con
     }
     renderSurface->GetShadingFrameBuffer()->Unbind();
 
-    /// @todo Render skinned meshes.
+    // Render skinned meshes.
+    renderSurface->GetShadingFrameBuffer()->BindWrite();
+    { PROFILE("Render skinned meshes");
+    { GPUPROFILE("Render skinned meshes", Video::Query::Type::TIME_ELAPSED);
+    { GPUPROFILE("Render skinned meshes", Video::Query::Type::SAMPLES_PASSED);
+
+        // Cull lights and update light list.
+        LightWorld(world, viewMatrix, projectionMatrix, viewProjectionMatrix);
+
+        // Push matricies and light buffer to the GPU.
+        renderer->PrepareSkinnedMeshRendering(viewMatrix, projectionMatrix);
+
+        // Render meshes
+        /// @todo Sort meshes after animation controller instead of
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetDirectionalLights' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)